packages/dara-components/dara/components/common/button.py

```diff
@@ -49,14 +49,16 @@ class Button(LayoutComponent):
 
     ```python
 
-    from dara.core.actions import NavigateTo
-    from dara.components.common import Button, ButtonStyle
+    from dara.core import action
+    from dara.components import Button, ButtonStyle
 
-    action = NavigateTo(url='/test')
+    @action
+    async def navigate_to(ctx: action.Ctx):
+        await ctx.navigate(url='/test')
 
     Button(
         'Click',
-        onclick=action,
+        onclick=navigate_to(),
```

Comment:
Same here and below, with a static url just construct `NavigateTo` object directly with it e.g. `onclick=NavigateTo(url)`. Otherwise we need to call to the backend to get the url

Reply:
fixed